src/commonMain/kotlin/exchange.dydx.abacus/functional/vault/Vault.kt

```diff
@@ -188,7 +212,29 @@ object VaultCalculator {
                 perpetualMarket = market,
             )
         }
-        return VaultPositions(positions = positions?.toIList())
+
+        if (vault.details?.totalValue != null) {
+            val usdcTotal = vault.details.totalValue - positions.sumOf { it.marginUsdc ?: 0.0 }
+
+            // add a usdc row
+            positions = positions + VaultPosition(
```

Comment:
Nit:  Duplication of line 172, so probably better to separate this out as a function.

Reply:
lol it's all duplicated though :) 

(adding a helper function now)